### DISCUSSION TITLE
See full hostname in dashboard

### DISCUSSION THREAD

**Discussion by JerryVDP** [plausible/analytics#2983] [Self-Hosted Support]
Hi,

First of all thank you for this amazing piece of software!

I have a website where users can create their own "mini sites" with a subdomain of the main website.

So let's say my website is `example.com` a user may register and create a `username.example.com` website.

All of those subdomains (and main domain) report to the same site ID.

My question is: how can I distinguish the different subdomains in my dashboard?

**Comment by clementmas** [comment]
I believe the current workaround is to create "roll-up views": https://plausible.io/docs/plausible-script#can-i-use-the-same-snippet-across-my-domain-and-subdomain

I'm also looking for a better way to distinguish the different domains from the dashboard and the API.
